QuickFIXn/Fields/Converters/DateTimeConverter.cs

```diff
@@ -9,260 +10,624 @@ namespace QuickFix.Fields.Converters;
 /// </summary>
 public static class DateTimeConverter
 {
-    public const int MicrosPerMillis = 1000;
-    public const int NanosPerMicro = 1000;
-    public const int TicksPerMicrosecond = 10;
     public const int NanosecondsPerTick = 100;
```

Comment:
This exists on TimeSpan these days so could also be deleted https://learn.microsoft.com/dotnet/api/system.timespan.nanosecondspertick?view=net-9.0#system-timespan-nanosecondspertick